src/in.txt

```diff
@@ -0,0 +1,25 @@
+student1;HISTORY;8,7,3;CULTURE;7,7,5,3;ENGLISH;9
```

Comment:
I suggest creating a resources folder that will keep such files as this instead of storing it in the root of project

Reply:
Data folder created